### PR TITLE
Dc/feature/create likes schema

### PR DESCRIPTION
**Pull request by orangeboy55555**
Merge branch to develop

### CODE REVIEW

File: backend/unit_tests/post-service-test.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ const sinon = require("sinon");
 const Post = require('../schema/post-schema');
 const mongoose = require('mongoose');
 const expect = require('chai').expect;
+require("dotenv").config();
+
+const useRealDatabase = false;
```
Comment (DaTanUmanitoba) [inline backend/unit_tests/post-service-test.js:8]: So, the service would be also tested on the real database?

File: backend/unit_tests/post-service-test.js [hunk 2]
```diff
@@ -170,14 +193,12 @@ describe('Post services and model', function () {
...
-            posts = [];
 
             const value = await services.getAllPosts();
             expect(value.length).to.equal(0);
         });
 
         it('should return one post', async function() {
-            posts = [];
 
             await services.createPost(new mongoose.mongo.ObjectID, '1', '1');
```
Comment (DaTanUmanitoba) [inline backend/unit_tests/post-service-test.js:203]: Is the new post added in the real database here? Then do we need to remove that after the test?

Comment (orangeboy55555) [reply backend/unit_tests/post-service-test.js:203]: That line shouldn't be there. I'll update it later.

**Comment by DaTanUmanitoba** [comment]
looks good to me.

**Pull request merged**